.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Docfx V3 Release
+on:
+  push:
+    branches: [ v3-release ]
+jobs:
+  build:
+    name: Release
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
I guess that depth 0 will make the generated version number `{height}` `1`?

Reply:
`fetch-depth`: 0 means fetch all commits